tests/unit_tests/tree_test.py

```diff
@@ -0,0 +1,39 @@
+from gopher import tree_search
+from gopher import enrichment
+
+
+def test_tree_mapping(generate_mapping):
+    """Test the tree mapping functionality maps the terms and returns the new mapping in a dictionary."""
+    mapping = generate_mapping
+    go_subset = ["a", "y", "z"]
+    mapped = tree_search.new_tree_map(mapping, go_subset)
+    assert isinstance(mapped, dict)
+
```

Comment:
**question:** Is `mapped` small enough that you could generate what you expect it to be it by hand? If so, you should test for an exact match.

Reply:
Yes, that is done!